<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroCliente</class>
 <widget class="QDialog" name="CadastroCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Cliente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/img/img/clientei.png</normaloff>:/img/img/clientei.png</iconset>
  </property>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>9</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>941</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layoutGridCC">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="layoutUniaoCC">
      <item>
       <layout class="QVBoxLayout" name="layoutCCTelefone">
        <item>
         <widget class="QLabel" name="labelCCTelefone">
          <property name="text">
           <string>Telefone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCCTelefone"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutCCCPF">
        <item>
         <widget class="QLabel" name="labelCCCPF">
          <property name="text">
           <string>CPF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCCCPF"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="layoutUniaoCCBotao">
      <item>
       <spacer name="spacerCCD">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCCCadastrar">
        <property name="text">
         <string>Cadastrar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacerCCE">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="layoutCCNome">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="layoutCCID">
          <item>
           <widget class="QLabel" name="labelCCID">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCCID"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelCCNome">
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCCNome"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="layoutCCEndereco">
      <item>
       <widget class="QLabel" name="labelCCEndereco">
        <property name="text">
         <string>Endere√ßo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCCEndereco"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="spacerCCDD">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditCCID</tabstop>
  <tabstop>lineEditCCNome</tabstop>
  <tabstop>lineEditCCEndereco</tabstop>
  <tabstop>lineEditCCTelefone</tabstop>
  <tabstop>lineEditCCCPF</tabstop>
  <tabstop>pushButtonCCCadastrar</tabstop>
 </tabstops>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
